src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -43,6 +40,10 @@ private static void setupPresetTests(Application application) {
      * @param application Application context.
      */
     private static void setupCommandTests(Application application) {
+        SelectTestFigureCompoundRectangleOptionListener selectTestFigureCompoundRectangleOptionListener =
+                new SelectTestFigureCompoundRectangleOptionListener(DriverFeature.getDriverManager());
+        application.addTest("Compound Rectangle", selectTestFigureCompoundRectangleOptionListener);
```

Comment:
Please load only command to CommandManager (SetCurrentCommand) without execution!